### DISCUSSION TITLE
Migrate seaweed 0.7 to new version

### DISCUSSION THREAD

**Discussion by simoncmchan** [seaweedfs/seaweedfs#2582] [General]
Hi,

we are using seaweed 0.7 (so old) and since the server is not enough space and i want to migrate to new server.
any method / cmd that i can do? now replication=001.

(PS: i also want to upgrade to latest version too, i just copy data files?)

Regards

**Comment by chrislusf** [comment]
If using filer, it is not supported.

If not, there are no difference from normal upgrade.
